packages/lesswrong/components/messaging/MessagesNewForm.tsx

```diff
@@ -59,7 +59,8 @@ const styles = (theme: ThemeType): JssStyles => ({
         }),
   },
   formButtonMinimalist: {
-    padding: "2px",
+    padding: "8px",
+    margin: "-6px -6px -6px 0",
```

Comment:
Because the input is ckeditor, we can't use `alignItems: 'center'` to align the placeholder with the button, so this is about as good as I can do to make it look aligned while keeping the padding around the placeholder the same